src/main/groovy/au/org/ala/ecodata/converter/SciStarterConverter.groovy

```diff
@@ -292,11 +268,11 @@ class SciStarterConverter {
                 "type"        : "projectArea"
         ]
 
-        switch (props.geometry.type) {
+        switch (geometry.type) {
             case 'MultiPolygon':
-                site.name = props.name;
+                site.name = project.regions_description;
                 // possible data loss here. converting multipolygon to polygon since biocollect/merit does not support it.
```

Comment:
Is this still true?  MERIT does support rendering multi-polyons, although maybe not via the editor UI but I doubt we want to be manually editing sci-starter polygons.

Reply:
BioCollect still doesn't support the rendering of multi-polygons - so i've left this is unchanged from the original. I thought about trying to magic it into one Polygon using JTS, but this seemed like this simplest approach